src/views/tables/PositionsTable/PositionsTriggersCell.tsx

```diff
@@ -223,10 +241,28 @@ Styled.Label = styled.div<{ warning?: boolean }>`
       background-color: var(--color-warning);
       color: var(--color-black);
     `}
+
+  ${({ hasOrders }) =>
+    hasOrders
+      ? css`
+          color: var(--color-text-1);
+          background-color: var(--color-layer-4);
+        `
+      : css`
+          color: var(--color-text-0);
+        `}
 `;
 
-Styled.Output = styled(Output)`
+Styled.Output = styled(Output)<{ value?: number }>`
   font: var(--font-mini-medium);
+  ${({ value }) =>
+    value
+      ? css`
```

Comment:
eh I wonder if it feels like this styling should live in `Output` // not a strong thought though